doc/man3/CMS_encrypt.pod

```diff
@@ -26,7 +27,8 @@ EVP_des_ede3_cbc() (triple DES) is the algorithm of choice for S/MIME use
 because most clients will support it.
 
 The algorithm passed in the B<cipher> parameter must support ASN1 encoding of
-its parameters.
+its parameters. If the cipher mode is GCM, then an AuthEnvelopedData structure
+containing MAC is used. Otherwise an EnvelopedData structure is used.
```

Comment:
Perhaps we should say other AEAD algorithms are not supported.

Reply:
Done. Not sure about the wording. Happy to change it if you prefer a different wording?

Reply:
The wording looks fine.